### PR TITLE
Specify default configuration instead of sideci_config

### PR DESCRIPTION
**Pull request by akito19**
We have https://github.com/actcat/sideci_config as our service's default configuration settings.
However, it is too old to run on our documents because the settings were published for our ancient systems.

Thus, I specify default settings not but sideci_config URL.

### CODE REVIEW

File: docs/tools/javascript/jshint.md [hunk 1]
```diff
@@ -19,9 +19,28 @@ To customize the configuration, use the standard `.jshintrc` or `.jshintignore`
 
 ## Default Configuration
 
-Sider uses its default configuration where there is no custom configuration preset:
+Sider uses the default configuration where there is no custom configuration preset. The configuration is here:
 
-* [Sider's configuration for jshintrc](https://github.com/actcat/sideci_config/blob/master/javascript/jshint/sideci_jshintrc)
+```json
```
Comment (ybiquitous) [inline docs/tools/javascript/jshint.md]: [suggestion] How about changing to `json5` for the syntax highlight language? The comments in JSON might be shown properly.

```suggestion
```json5
```

File: docs/tools/javascript/jshint.md [hunk 2]
```diff
@@ -19,9 +19,28 @@ To customize the configuration, use the standard `.jshintrc` or `.jshintignore`
...
+{
+  // TODO: Enable Enforcing options
```
Comment (ybiquitous) [inline docs/tools/javascript/jshint.md]: [question] Is the `TODO` comment necessary?

File: docs/tools/javascript/jshint.md [hunk 3]
```diff
@@ -19,9 +19,28 @@ To customize the configuration, use the standard `.jshintrc` or `.jshintignore`
...
+
+  // Environments
+  "jquery": true,
+  "browser": true,
+
+  "esversion": 2015
+}
+```
+
+In addition, `public/js/**/*.js` and `**/*.min.js` files are ignored.
```
Comment (ybiquitous) [inline docs/tools/javascript/jshint.md]: [suggestion] How about describing by using `.jshintignore` like this?

---

In addition, the following `.jshintignore` is used by default:

```
public/js/**/*.js
**/*.min.js
```

File: docs/tools/php/phpmd.md [hunk 4]
```diff
@@ -45,9 +45,45 @@ linter:
...
+         xsi:noNamespaceSchemaLocation="
+                     http://pmd.sf.net/ruleset_xml_schema.xsd">
+  <description>
+    This rule set is recommended by SideCI.
+    It contains only generic rules in all projects.
+  </description>
+
+  <!--
+  <rule ref="rulesets/cleancode.xml" />
+  -->
```
Comment (ybiquitous) [inline docs/tools/php/phpmd.md]: [suggestion] I think the comment in the configuration isn't necessary for users, so how about removing them?

File: docs/tools/php/phpmd.md [hunk 5]
```diff
@@ -45,9 +45,45 @@ linter:
...
+    <exclude name="CamelCasePropertyName"></exclude>
+    <exclude name="CamelCaseMethodName"></exclude>
+    <exclude name="CamelCaseParameterName"></exclude>
+    <exclude name="CamelCaseVariableName"></exclude>
+  </rule>
+  <rule ref="rulesets/design.xml" />
+  <!--
+    We think Naming rule is not important. So, we exclude it.
+  <rule ref="rulesets/naming.xml" />
+  -->
```
Comment (ybiquitous) [inline docs/tools/php/phpmd.md]: It's also same.

File: docs/tools/go/gometalinter.md [hunk 6]
```diff
@@ -33,9 +33,29 @@ If `glide.yaml` contains a dependency to a library in a private repository, plea
 
 ## Default Configuration
 
-If your `sider.yml` does not contain `config` option, Sider will use the default configuration. The default configuration is available in our repository.
-
-* [Sider's configuration for Go Meta Linter](https://github.com/actcat/sideci_config/blob/master/go/gometalinter/gometalinter.json)
+If your `sider.yml` does not contain `config` option, Sider will use the default configuration as follows:
```
[the remark this answers is not in the document]
Comment (akito19) [reply docs/tools/go/gometalinter.md]: I would not like to touch gometalinter docs....

[the remark this answers is not in the document]
Comment (ybiquitous) [reply docs/tools/go/gometalinter.md]: Why? Just adding an internal link...

[the remark this answers is not in the document]
Comment (akito19) [reply docs/tools/go/gometalinter.md]: I feel emptiness to maintain docs in spite of its deprecation, but I'm fixing it without my heart...

[the remark this answers is not in the document]
Comment (ybiquitous) [reply docs/tools/go/gometalinter.md]: I see 👍

File: docs/tools/javascript/jshint.md [hunk 7]
```diff
@@ -19,9 +19,29 @@ To customize the configuration, use the standard `.jshintrc` or `.jshintignore`
...
+  "jquery": true,
+  "browser": true,
+
+  "esversion": 2015
+}
+```
+
+In addition, we uses `.jshintignore` to default with following setting:
+
+```json5
```
Comment (ybiquitous) [inline docs/tools/javascript/jshint.md]: `json5` seems to be unnecessary here. 😅

File: docs/tools/php/phpmd.md [hunk 8]
```diff
@@ -45,9 +45,38 @@ linter:
...
+<?xml version="1.0"?>
+<ruleset name="SideCI Recommended rule set"
+         xmlns="http://pmd.sf.net/ruleset/1.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://pmd.sf.net/ruleset/1.0.0
+                     http://pmd.sf.net/ruleset_xml_schema.xsd"
+         xsi:noNamespaceSchemaLocation="
+                     http://pmd.sf.net/ruleset_xml_schema.xsd">
+  <description>
+    This rule set is recommended by SideCI.
```
Comment (ybiquitous) [inline docs/tools/php/phpmd.md]: I find 2 typos 😓 

```suggestion
    This ruleset is recommended by Sider.
```

File: docs/tools/php/phpmd.md [hunk 9]
```diff
@@ -45,9 +45,38 @@ linter:
...
 ## Default Configuration
 
-If you leave the `rule` option undefined in `sider.yml`, Sider runs PHPMD with our recommended settings. The recommended settings are available in our GitHub repository:
-
-* [Sider recommended settings for PHPMD](https://github.com/actcat/sideci_config/blob/master/php/phpmd/sideci_config.xml)
+If you leave the `rule` option undefined in `sider.yml`, Sider runs PHPMD with the following configuration to default:
+
+```xml
+<?xml version="1.0"?>
+<ruleset name="SideCI Recommended rule set"
```
Comment (ybiquitous) [inline docs/tools/php/phpmd.md]: Also here 😅 

```suggestion
<ruleset name="Sider recommended ruleset"
```

File: docs/tools/python/flake8.md [hunk 10]
```diff
@@ -27,9 +27,28 @@ The latest versions of Python 2 or Python 3 can be used.
 
 Sider provides a default configuration for Flake8. If your repository does not include `.flake8`, `setup.cfg` or `tox.ini`, our default configuration will be used.
 
-Our default configuration is available here:
+Our default configuration is here:
 
-* [Sider's configuration for Flake8](https://github.com/actcat/sideci_config/blob/master/python/flake8/sideci_config.ini)
+```
```
Comment (ybiquitous) [inline docs/tools/python/flake8.md]: The syntax highlighting for `init` files seems to be available here. 👀 

```suggestion
```init
```

File: docs/tools/javascript/jshint.md [hunk 11]
```diff
@@ -19,9 +19,29 @@ To customize the configuration, use the standard `.jshintrc` or `.jshintignore`
...
+  "jquery": true,
+  "browser": true,
+
+  "esversion": 2015
+}
+```
+
+In addition, we uses `.jshintignore` to default with following setting:
+
+```json
```
Comment (ybiquitous) [inline docs/tools/javascript/jshint.md]: `. jshintignore` is not a JSON file, so this `json` is unneeded. 💦 

```diff
-```json
+```
```

Comment (akito19) [reply docs/tools/javascript/jshint.md]: 🙃

**Pull request merged**